### PR TITLE
Use inputs instead of thunks through airline-demo

### PR DESCRIPTION
**Pull request by schrockn**
Also consolidated configs to files and some other cleanup.

### CODE REVIEW

File: python_modules/airline-demo/airline_demo/solids.py [hunk 1]
```diff
@@ -162,29 +161,6 @@ def thunk(info):
...
-        info (ExpectationExecutionInfo)
-
-    Returns:
-        list:
-            The config value passed to the solid.
-    '''
-    return info.config
-
-
 @solid(
```
Comment (mgasner) [inline python_modules/airline-demo/airline_demo/solids.py]: following this line of though, presumably we eventually have an input type here which takes config

File: python_modules/airline-demo/airline_demo/solids.py [hunk 2]
```diff
@@ -447,26 +426,15 @@ def unzip_file(
...
-                'input_csv': Field(types.Path, description='', default_value='', is_optional=True),
-            }
-        )
-    ),
-    inputs=[InputDefinition(
-        'input_csv',
-        types.Path,
-        description='',
-    )],
+    inputs=[InputDefinition('input_csv', types.Path)],
```
Comment (mgasner) [inline python_modules/airline-demo/airline_demo/solids.py]: incredible how much cleaner this is

File: python_modules/airline-demo/airline_demo/test/test_pipelines.py [hunk 3]
```diff
@@ -16,327 +19,37 @@
 @pytest.mark.db
 @pytest.mark.py3
 def test_pipelines():
-    now = datetime.datetime.utcnow()
-    timestamp = now.strftime('%Y_%m_%d_%H_%M_%S')
+    download_config_object = load_yaml_from_globs(
```
Comment (mgasner) [inline python_modules/airline-demo/airline_demo/test/test_pipelines.py]: we may want to give `load_yaml_from_globs` a different name if it's going to be the primary public API we promote for programmatically loading config

File: python_modules/airline-demo/airline_demo/solids.py (hunk 2 above)
Comment (schrockn) [reply python_modules/airline-demo/airline_demo/solids.py]: 👍🏻

File: python_modules/airline-demo/airline_demo/test/test_pipelines.py (hunk 3 above)
Comment (schrockn) [reply python_modules/airline-demo/airline_demo/test/test_pipelines.py]: ya definitely not a public api

File: python_modules/airline-demo/airline_demo/solids.py (hunk 1 above)
Comment (schrockn) [reply python_modules/airline-demo/airline_demo/solids.py]: thunks should be totally unnecessary if the type is an input which is Configurable

**Pull request merged**